Source/automap.cpp

```diff
@@ -531,10 +531,12 @@ WORD GetAutomapType(int x, int y, BOOL view)
 	}
 
 	rv = automaptype[(BYTE)dungeon[x][y]];
-	if (rv == 7
-	    && GetAutomapType(x - 1, y, FALSE) & (MAPFLAG_HORZARCH << 8)
-	    && GetAutomapType(x, y - 1, FALSE) & (MAPFLAG_VERTARCH << 8)) {
-		rv = 1;
+	if (rv == 7) {
+		if ((GetAutomapType(x - 1, y, FALSE) >> 8) & MAPFLAG_HORZARCH) {

```

Comment:
Theres actually a variable missing here, which explains this one. Have it on my local cleanups but havent uploaded yet.

Reply:
> Theres actually a variable missing here, which explains this one. Have it on my local cleanups but havent uploaded yet.

I'll never seize to be amazed at the insights you've built up in that mind of yours Andi. There is nothing too minute of a detail, for which you can't add an extra nuance! *hugging your brain and heart* (because that's a thing you can do).

Reply:
`hugging your brain and heart (because that's a thing you can do).` - only once :)